src/components/entity/ha-state-label-badge.html

```diff
@@ -43,6 +43,7 @@
       image='[[computeImage(state)]]'
       label='[[computeLabel(localize, state, timerTimeRemaining)]]'
       description='[[computeDescription(state)]]'
+      label-is-state$='[[!state.attributes.unit_of_measurement]]'
```

Comment:
The name `is-state` is ambiguous. They are all states.

However, another reason to rename it is that it is violating the seperation of concern that we created. The state-label-badge component knows it's a state. The label-badge doesn't, it's just a badge showing stuff. So instead I think that we should allow overriding the label text-transform with a CSS property

Reply:
I first tried 'label-is-unit' but I could not cast `state.attributes.unit_of_measurement` to a boolean by using `!!` (this is expected, as stated in Polymer docs)
Setting an attribute like that is a lot easier than setting a class (for which you'd need to have some extra javascript)

Agree on the css-variable thing. The ha-label-badge should not need to know what (for example) a unit-of-measurement even is, and what it should do with it.